app/views/catarse_bootstrap/pages/hello.html.slim

```diff
@@ -3,13 +3,13 @@
 
 .hero-hello
   .w-embed.w-hidden-medium.w-hidden-small.w-hidden-tiny
-    video autoplay="" loop="" style=("width:100%; height: auto; position:absolute; z-index: -1;") 
+    video poster="/assets/hero-hello.jpg" autoplay="" loop="" style=("width:100%; height: auto; position:absolute; z-index: -1;") 
```

Comment:
Should use image_path helper function:
http://apidock.com/rails/ActionView/Helpers/AssetUrlHelper/image_path